frontend/src/pages/distributedWorkloads/global/const.ts

```diff
@@ -0,0 +1,2 @@
+export const distributedWorkloadsPageTitle = 'Workload Metrics';
+export const distributedWorkloadsPageDescription = 'TODO page description here';
```

Comment:
Don't put these in constant files, just inline the text on screen (unless you intend to repeat the same usage and constants can work out well (like Pipelines).

Reply:
Ah yeah, in an earlier commit I had a different `<ApplicationsPage>` rendered in each tab component so this made more sense, but probably not now. Good call